lib/ash_json_api/json_schema/open_api.ex

```diff
@@ -200,6 +200,7 @@ if Code.ensure_loaded?(OpenApiSpex) do
     defp resource_attributes(resource) do
       resource
       |> Ash.Resource.Info.public_attributes()
+      |> Enum.concat(Ash.Resource.Info.public_calculations(resource))
```

Comment:
```elixir
defp filterable?(%Ash.Resource.Calculation{type: type, calculation: {module, _opts}}, _) do
    !Ash.Type.embedded_type?(type) && function_exported?(module, :expression, 2)
  end
```

ash_graphql uses the above logic to check if a calculation should be in the list of sortable fields. We should probably make them both use the same logic from core though tbh

Reply:
Just added a filter for what's sortable using `Ash.Resource.Info.sortable?/2`. In the case of filtering, this clause in particular just specifies what the return object looks like, so I think we're ok to skip the `filterable?` check here. I haven't added calculations to the filter params just yet.